src/Sulu/Bundle/AdminBundle/Resources/js/containers/Form/fields/TextArea.js

```diff
@@ -58,6 +65,7 @@ export default class TextArea extends React.Component<FieldTypeProps<?string>> {
                 onBlur={onFinish}
                 onChange={onChange}
                 onFocus={this.handleFocus}
+                rows={rows}
```

Comment:
```suggestion
                rows={rows ? parseInt(rows) : undefined}
```

Same handling as for: maxCharacters